ports/esp8266/ets_alt_task.c

```diff
@@ -142,34 +139,27 @@ bool ets_loop_iter(void) {
     extern uint32_t pend_flag_noise_check;
     uint32_t *sw_wdt = &pend_flag_noise_check - 6;
 
-    //static unsigned cnt;
     bool progress = false;
     for (volatile struct task_entry *t = emu_tasks; t < &emu_tasks[MP_ARRAY_SIZE(emu_tasks)]; t++) {
         if (!ets_loop_dont_feed_sw_wdt) {
             system_soft_wdt_feed();
         }
         ets_intr_lock();
-        //printf("etc_loop_iter: "); dump_task(t - emu_tasks + FIRST_PRIO, t);
         if (t->i_get != t->i_put) {
             progress = true;
-            //printf("#%d Calling task %d(%p) (%x, %x)\n", cnt++,
-            //    t - emu_tasks + FIRST_PRIO, t->task, t->queue[t->i_get].sig, t->queue[t->i_get].par);
```

Comment:
This one should be converted to `DEBUG_printf` style, but that's a bit more involved here.... so probably best for a separate commit (which I could do).